Gemfile

```diff
@@ -1,43 +1,23 @@
-source 'https://rubygems.org'
-
-git_source(:github) do |repo_name|
-  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
-  "https://github.com/#{repo_name}.git"
-end
```

Comment:
This is actually included by default in bundler these days. The theory behind it is you can source gems from GitHub directly going `github: foo/bar` rather than `git: https://git.github.com/foo/bar.git`. It does look ugly though, so happy to see it gone.

Reply:
Yeah. I have a habit of removing it from the Gemfile, only adding it back if needed.